chore(release): add release notes for 2.30.3 (#2180)
armory/docs: content/en/continuous-deployment/release-notes/rn-armory-spinnaker/armoryspinnaker_v2-30-3.md

**Affected versions**: Armory CD 2.30.0 and later

### Clouddriver and Spring Cloud

The Spring Boot version has been upgraded, introducing a backwards incompatible change to the way configuration is loaded in Spinnaker. Users will need to set the ***spring.cloud.config.enabled*** property to ***true*** in the service settings of Clouddriver to preserve existing behavior. All of the other configuration blocks remain the same.

**Affected versions**: Armory CD 2.30.0 and later

### SpEL expressions and artifact binding

There is an issue where it appears that SpEL expressions are not being evaluated properly in artifact declarations (such as container images) for events such as the Deploy Manifest stage. What is actually happening is that an artifact binding is overriding the image value.

**Workaround**:

2.27.x or later: Disable artifact binding by adding the following parameter to the stage JSON: `enableArtifactBinding: false`. This setting only binds the version when the tag is missing, such as `image: nginx` without a version number.

**Affected versions**: Armory CD 2.27.x and later

## Deprecations

> Reference [Feature Deprecations and end of support](https://docs.armory.io/continuous-deployment/feature-status/deprecations/)

Reviewer: ```suggestion
Reference [Feature Deprecations and end of support]({{< ref "continuous-deployment/feature-status/deprecations/" >}})
```

    version: 2.30.3
  echo:
    commit: 2ef241fd3da29fb70cdb05432d022f0edd752d51
    version: 2.30.3
  fiat:
    commit: 5d44e1f53d2f33b17f8decfb057a18cfe4b6fa08
    version: 2.30.3
  front50:
    commit: 5d9bb31f65f96087be30ce96cea1b6d481a6bef4
    version: 2.30.3
  gate:
    commit: 04598366642300d6f028df33b6206b5ce75f1038
    version: 2.30.3
  igor:
    commit: f3d890427c79b7db6cf5fcb681e30542df97ff7c
    version: 2.30.3
  kayenta:
    commit: 1d2f5193ec681b5122fe7c34da6bbd569da8e0b8
    version: 2.30.3
  monitoring-daemon:
    commit: null
    version: 2.26.0
  monitoring-third-party:
    commit: null
    version: 2.26.0
  orca:
    commit: ba62a86ef3af0007506b589a4272e3c2e03de00b
    version: 2.30.3
  rosco:
    commit: a2b4662a40281e553de2182b680d5fd7e6bc3955
    version: 2.30.3
  terraformer:
    commit: 249e7be18074af100212ddd554d9fb35afd65873
    version: 2.30.3
timestamp: "2023-09-29 15:15:45"
version: 2.30.3
</code>
</pre>
</details>

### Armory

#### Armory Gate - 2.30.2...2.30.3

  - chore(cd): update base service version to gate:2023.09.01.01.26.23.release-1.30.x (#608)
  - chore(cd): update base service version to gate:2023.09.07.17.47.28.release-1.30.x (#612)

>   - chore(ci): removed aquasec scan action (#616) (#622)

Reviewer 2: Not relevant to customers as it's an internal build tooling change
```suggestion
```

#### Armory Rosco - 2.30.2...2.30.3

>   - chore(ci): removed aquasec scan action (#565) (#568)

Reviewer 2: ```suggestion
```

#### Armory Orca - 2.30.2...2.30.3

  - chore(cd): update base orca version to 2023.09.07.17.59.31.release-1.30.x (#693)

>   - chore(ci): removed aquasec scan action (#695) (#706)

Reviewer 2: ```suggestion
```

  - chore(cd): update base orca version to 2023.09.26.15.03.20.release-1.30.x (#721)

#### Armory Fiat - 2.30.2...2.30.3

>   - chore(ci): removed aquasec scan action (#523) (#526)

Reviewer 2: ```suggestion
```

#### Armory Echo - 2.30.2...2.30.3

  - chore(cd): update base service version to echo:2023.09.07.17.49.13.release-1.30.x (#616)

>   - chore(ci): removed aquasec scan action (#618) (#622)

Reviewer 2: ```suggestion
```

#### Dinghy™ - 2.30.2...2.30.3

>   - chore(ci): removed aquasec scan action (#489) (#492)
>   - chore(ci): removing aquasec scans for any push (#497) (#499)

Reviewer 2: ```suggestion
```

  - chore(log): Bumped up dinghy and plank versions. (backport #490) (#505)

#### Armory Kayenta - 2.30.2...2.30.3

>   - chore(ci): removed aquasec scan action (#469) (#471)

Reviewer 2: ```suggestion
```

#### Armory Clouddriver - 2.30.2...2.30.3

  - chore(cd): update base service version to clouddriver:2023.08.29.09.35.30.release-1.30.x (#945)
  - chore(cd): update base service version to clouddriver:2023.09.07.19.02.07.release-1.30.x (#953)
  - chore(cd): update base service version to clouddriver:2023.09.07.22.04.28.release-1.30.x (#955)
  - chore(cd): update base service version to clouddriver:2023.09.08.02.48.03.release-1.30.x (#958)
  - chore(cd): update base service version to clouddriver:2023.09.08.03.51.25.release-1.30.x (#959)
  - chore(cd): update base service version to clouddriver:2023.09.08.04.42.03.release-1.30.x (#962)
  - chore(cd): update base service version to clouddriver:2023.09.08.05.40.38.release-1.30.x (#963)
  - chore(cd): update base service version to clouddriver:2023.09.08.14.33.30.release-1.30.x (#965)
  - chore(cd): update base service version to clouddriver:2023.09.16.22.42.11.release-1.30.x (#976)
  - chore(cd): update base service version to clouddriver:2023.09.20.19.03.35.release-1.30.x (#980)

>   - chore(ci): removed aquasec scan action (#971) (#983)

Reviewer 2: ```suggestion
```

  - fix: CVE-2023-37920 (#977) (#992)

#### Armory Front50 - 2.30.2...2.30.3

  - chore(cd): update base service version to front50:2023.09.07.17.50.48.release-1.30.x (#588)

>   - chore(ci): removed aquasec scan action (#590) (#594)

Reviewer 2: ```suggestion
```

  - chore(cd): update base service version to front50:2023.09.25.18.58.31.release-1.30.x (#601)
  - fix(dependencies): match google cloud storage version to this set in OSS, and bump OSS front50 version. (#602)

#### Terraformer™ - 2.30.2...2.30.3

>   - chore(ci): removing aquasec scans on push (#517) (#519)
>   - chore(ci): removed aquasec scan action (#510) (#512)

Reviewer 2: ```suggestion
```

#### Armory Deck - 2.30.2...2.30.3

  - chore(cd): update base deck version to 2023.0.0-20230918174859.release-1.30.x (#1342)
  - chore(cd): update base deck version to 2023.0.0-20230920215154.release-1.30.x (#1343)

>   - chore(ci): removed aquasec scan action (#1340) (#1345)
>   - chore(ci): removing docker build and aquasec scans (#1350)

Reviewer 2: ```suggestion
```

#### Armory Igor - 2.30.2...2.30.3

  - chore(cd): update base service version to igor:2023.09.07.17.50.00.release-1.30.x (#491)

>   - chore(ci): removed aquasec scan action (#495) (#508)

Reviewer 2: ```suggestion
```
